apps/AEPSampleAppNewArchEnabled/app/(tabs)/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Image, StyleSheet, Platform } from 'react-native';
```

Comment:
Q: Why is `(tabs)` used as the directory name? Is it some kind of a naming convention?

Reply:
have removed already